example/lib/peripheral_list/components/peripheral_item.dart

```diff
@@ -49,24 +51,15 @@ class PeripheralItem extends StatelessWidget {
   void _onRowTap(NavigationBloc navigationBloc) {
     navigationBloc.add(NavigateToPeripheralDetails(peripheral: _peripheral));
   }
-  
+
   String _formatRssi(int rssi) {
     return '${rssi ?? '-'} dbm';
   }
 
   Color _colorForRssi(int rssi) {
-    if (rssi == null) return Colors.grey;
-
-    switch (parseRssi(rssi)) {
-      case SignalLevel.high:
-        return Colors.green;
-      case SignalLevel.medium:
-        return Colors.orange;
-      case SignalLevel.low:
-        return Colors.red;
-      default:
-        return Colors.grey;
-    }
+    return rssi == null
+        ? Colors.grey
```

Comment:
Is the point of having `ColorManager.colorForSignalLevel()` not to remove any checks here?

Reply:
Missed that. Fixed